contracts/tests/HotfixClaimRevenue.t.sol

```diff
@@ -0,0 +1,334 @@
+        vm.selectFork(mainnetFork);
+
+        borrower = 0xf44B95991CaDD73ed769454A03b3820997f00873; 
+        lender = address(10);
+        arbiter = address(this);
+
+        moduleFactory = new ModuleFactory();
+        lineFactory = new LineFactory(
+            address(moduleFactory),
+            arbiter,
+            ORACLE,
+            payable(ZERO_EX)
+        );
+
+        ILineFactory.CoreLineParams memory coreParams = ILineFactory
+            .CoreLineParams({
+                borrower: borrower,
+                ttl: ttl,
+                cratio: 1000,
+                revenueSplit: 90
+            });
+    
+        vm.startPrank(borrower);
+        lineAddress = lineFactory.deploySecuredLineWithConfig(coreParams);
+        vm.stopPrank();
+
+        line = SecuredLine(payable(lineAddress));
+        escrow = line.escrow();
+        spigot = line.spigot();
+
+    }
+
+    function _createAndFundLine(uint256 loanAmount, uint256 collateral) internal {
+
+        // enable collateral
+        escrow.enableCollateral(SNX);
+
+        // add a revenue stream
+        ISpigot.Setting memory setting = ISpigot.Setting({
+            ownerSplit: revenueSplit, // 90
+            claimFunction: bytes4(0),
+            transferOwnerFunction: bytes4("1234")
+        });
+
+        line.addSpigot(BORROWER_REVENUE_EOA, setting);
+
+        // ensure balances are sufficient
+        if (IERC20(SNX).balanceOf(borrower) < collateral) {
+            deal(SNX, borrower, collateral);
+        }
+        deal(DAI, lender, loanAmount);
+        deal(USDC, borrower, 100e6);
+        deal(USDC, BORROWER_REVENUE_EOA, 100e6);
+
+        emit log_named_uint("Mo SNX balance", IERC20(SNX).balanceOf(borrower));
+        emit log_named_uint("Mo DAI balance", IERC20(DAI).balanceOf(borrower));
+        emit log_named_uint("Mo USDC balance", IERC20(USDC).balanceOf(borrower));
+
+        // approve tokens, add collateral, and propose position
+        vm.startPrank(borrower);
+        IERC20(SNX).approve(address(escrow), MAX_INT);
+
+        escrow.addCollateral(collateral, SNX);
+
+        line.addCredit(
+            dRate,
+            fRate,
+            loanAmount,
+            DAI,
+            lender
+        );
+        vm.stopPrank();
+
+        // accept position
+        vm.startPrank(lender);
+        IERC20(DAI).approve(lineAddress, MAX_INT);
+        line.addCredit(
+            dRate,
+            fRate,
+            loanAmount,
+            DAI,
+            lender
+        );
+        vm.stopPrank();
+
+    }
+
+
+    // @note: This test replicates an accounting bug discovered in _claimRevenue in the SpigotLib
+    // contract that didn't take into account the operatorToken balance when claiming revenue
+    // @note: mainnet contract: 0x6e3a81f41210d45a2bbbbad00f25fd96567b9af2
+    function test_accounting_for_multiple_claim_revenue_invocations() external {
```

Comment:
can u rename to something like `test_reproduce_bug_claim_revenue_multiple_push_payments_accounting`